sqlengine/postgres_engine_test.go

```diff
@@ -490,7 +490,7 @@ var _ = Describe("PostgresEngine", func() {
 	})
 
 	Describe("Extensions", func() {
-		It("can create extensions", func() {
+		It("can create and drop extensions", func() {
```

Comment:
Could do with a pair of `By("...")` lines here, to break up the test and make it easier to follow and debug